azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/configurations/DeploymentType.java

```diff
@@ -13,7 +13,8 @@
 public enum DeploymentType {
     MS_DEPLOY(ConstantValues.MS_DEPLOY_VALUE),
     ZIP(ConstantValues.ZIP_VALUE),
-    FTP(ConstantValues.FTP_VALUE);
+    FTP(ConstantValues.FTP_VALUE),
+    UNKNOWN(null);
```

Comment:
There is no need for this.

Reply:
The above unknown is not necessary, but this one is required for default return value, I have to have some value to switch in DeployMojo.